### ISSUE TITLE
zdict

### ISSUE THREAD

**Issue by axel7083** [kuronosuFear/MarioKartTourTools#1]
In the latest version there isn't "zdict" file in the _nabe_ directory. I tried to use as Dictionnary files such as "_catalog.bytes" but they didn't work.

Do you know where the dictionnary is located ?

**Comment by kuronosuFear** [comment]
The file named "2ff16e1f.zst" is the encrypted zdict file. If you want to have the decrypted dictionary, you need to get it via memory dumps or by hooking the function "jp.co.nintendo.nabe.AndroidUtil.SetDictionary".

**Comment by awesomebytes** [comment]
@kuronosuFear I've just decompressed a couple of zst files from a dump of MKT 1.0.2 for curiosity. I found (by just doing `strings` on them) that they are "UnityFS" files. I don't know much about Unity (neither about Android, really, even thought I got to the disassembly of the APK). What do these files actually contain? Characters/Karts/etc 3D files?

I'm trying to find if there is somewhere where the drivers/karts/gliders information (as in, which can be used in which track) and maybe their specific stats can be accessed. Maybe all this info is taken just from on-line queries from some API... and if so I'd be interested in seeing if it can be intercepted, but that's probably another story.

Anyways, if you have any info to enlighten me, that would be great. Also don't doubt to contact me via another means if you think it wouldn't be good to say it here :)

Also thanks for uploading this project, I had fun discovering this zstd compression algorithm and playing around. Have a nice day!

Also, is there a rather trivial way of getting a decrypted dictionary? That's pure curiosity.
